lib/data/coding_profile_fields.csv

```diff
@@ -0,0 +1,4 @@
+Skills/Languages,text_area,,What tools and languages are you most experienced with? Are you specialized or more of a generalist?
```

Comment:
I added this file here so we can use it in `db/seeds.rb`, so our team members can easily get a profile as close to the current DEV one as possible. The file is tiny and we have other things in `lib/` that are DEV specific, e.g. old data update scripts.